scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -1161,32 +1176,39 @@ private Boolean isExecutionCalendarGIDefined(String jobDescriptor) {
         NodeList list = root.getChildNodes();
         for (int i = 0; i < list.getLength(); i++) {
             Node node = list.item(i);
-            if (node.getNodeName() == "info" && node.hasAttributes()) { // node.getNodeType() == Node.ELEMENT_NODE
+            if (node.getNodeName() == "info" && node.hasAttributes()) {
                 NamedNodeMap attributes = node.getAttributes();
                 for (int j = 0; j < attributes.getLength(); j++) {
                     Node attribute = attributes.item(j);
                     if (attribute.getNodeType() == Node.ATTRIBUTE_NODE && attribute.getNodeName().equals("name") &&
                         attribute.getNodeValue().equals("EXECUTION_CALENDARS")) {
                         exists = true;
                     }
-                    if (attribute.getNodeType() == Node.ATTRIBUTE_NODE && attribute.getNodeName().equals("value") &&
-                        exists) {
-                        if (attribute.getNodeValue() != null) {
+                    if (isAttributeExecCalendarValueDefined(attribute, "value") && exists) {
+                        if (attribute.getNodeValue() != null && attribute.getNodeValue().toString() != "" && exists) {
                             executionCalendarDefined = true;
-                        } else {
-                            displayErrorMessage("EXECUTION_CALENDAR value is empty.");
+                            if (attribute.getNodeValue() != "") {
```

Comment:
Same as the comment above (reference comparison)

Reply:
Addressed.